lib/ansible/modules/database/postgresql/postgresql_db.py

```diff
@@ -88,7 +88,18 @@
       - The database state
```

Comment:
Please add a comment on the `description` to say dump and import where added in 2.3

Also is it import or restore?